client/java/src/main/java/org/datajunction/client/Main.java

```diff
@@ -0,0 +1,7 @@
+package org.datajunction.client;
```

Comment:
Since the site is [datajunction.io](https://datajunction.io/), this should be `io.datajunction.client`. The directory needs to be changed as well and also the group in the build.gradle would need to be `io.datajunction`.